docker/README.md

```diff
@@ -100,19 +101,25 @@ $ docker push polarlightning/lnd:<version>
 
 **Building the image**
 
+Core Lightning requires building the arm64 image using a separate Dockerfile, so docker buildx cannot be used. We must build two images separately them merge them into a single multi-arch image using docker manifest.
+
 ```sh
 $ cd clightning
-$ docker build --build-arg CLN_VERSION=<version> -t polarlightning/clightning:<version> .
-```
+# build amd64 image (note: this takes a long time on ARM machines)
+$ docker build --platform linux/amd64 --build-arg CLN_VERSION=<version> -t polarlightning/clightning:<version>-amd64 .
+$ docker push polarlightning/clightning:<version>-amd64
 
-Replace `<version>` with the desired c-lightning version (ex: `0.8.0`).
-
-**Push to Docker Hub**
+# build arm64 image (note: this takes a long time on x86/x64 machines)
+$ docker build --platform linx/arm64 --build-arg CLN_VERSION=<version> -t polarlightning/clightning:<version>-arm64 -f Dockerfile.arm64 .
```

Comment:
I am doing one final round of builds for all nodes and discovered this small typo. Can you change `linx/arm64` to `linux/arm64`?